cpp/arcticdb/storage/test/in_memory_store.hpp

```diff
@@ -20,6 +20,7 @@
 #include <vector>
 
 namespace arcticdb {
+    /* In-memory implementation of Store for testing purposes. */
```

Comment:
Probably worth noting that a lot of methods are not implemented for this store (although @joe-iddon might be changing that soon)